src/modules/dotnet-publish.ts

```diff
@@ -27,40 +40,75 @@ const DotnetPublish = {
         return `Publishes the dotnet solution from the root of the project (via ${this.cmd()})`;
     },
     getOptions(): OptionStringBuilder {
-        return new OptionStringBuilder("publish", "p");
+        return Options.Publish;
     },
     /**
      * Runs a publish of the dotnet solution to the local file system
      * @param {string} absoluteOutputDir Optional absolute path of release output directory. If not provided,
      * defaults to dotnet solution's 'release' directory
      */
     run(absoluteOutputDir?: string) {
+        DotnetPath.solutionPathOrExit();
```

Comment:
I realize this function is not a part of this PR, but it's name is a little confusing to me.  Is this suggesting that this function will return the solution path if there is one, otherwise it'll exit?  If the name isn't clarified, I would consider adding a comment here to explain what this is doing.  Seems like a bandaid... would suggest refactoring if able to.

Reply:
See my comment below 👇 But yes, it's basically a wrapper around the `DotnetPath.solutionPath()` function that handles the null checking & error exit if not found. It was pretty common logic in the dotnet-related modules that we always expected to have the solution/project path available before continuing.